.github/workflows/images.yaml

```diff
@@ -25,6 +25,9 @@ jobs:
     strategy:
```

Comment:
On line 23 we will need to distinguish between `environment: ci` and `environment: release`. The "release" environment should only happen on push tags and the "ci" environment for everything else.

Reply:
Why would you need a split for the environment?

If it needs a split we need to include the env name in the matrix. Let me know, I will update.

Reply:
Updated see last commit.